pallets/ctype/src/mock.rs

```diff
@@ -81,13 +77,13 @@ pub mod runtime {
 	impl frame_system::Config for Test {
 		type RuntimeOrigin = RuntimeOrigin;
 		type RuntimeCall = RuntimeCall;
-		type Index = u64;
-		type BlockNumber = u64;
+		type Block = Block;
+		type Nonce = u32;
```

Comment:
Also here. I would keep them `u64` as they were before, if there's no strong reason to change them. We could do so in a different PR.

Reply:
[74a1a67](https://github.com/KILTprotocol/kilt-node/pull/559/commits/74a1a674b19b7e9ebe95d1832d8ef41fbeb3b8e1)